### ISSUE TITLE
Conda install fails - missing dependencies

### ISSUE THREAD

**Issue by davismcc** [CGATOxford/UMI-tools#60]
Hi there

Nice work on this tool - unfortunately installation on Mac OS X Sierra failed for me due to missing dependencies that were unstated:
```
$ conda install -c https://conda.anaconda.org/toms umi_tools
Fetching package metadata .........
Solving package specifications: .


PackageNotFoundError: Package not found: '' Dependencies missing in current osx-64 channels:
  - umi_tools -> bcftools ==1.3
  - umi_tools -> htslib >=1.3
  - umi_tools -> pysam ==0.8.4
  - umi_tools -> samtools >=1.3

Close matches found; did you mean one of these?

    htslib: tblib, html5lib
    samtools: apptools, umi_tools

You can search for packages on anaconda.org with

    anaconda search -t conda samtools

(and similarly for the other packages)

```

Perhaps you could list these dependencies in the installation instructions and link to installation instructions for these tools?

Best
Davis

**Comment by davismcc** [comment]
To complete this thought: installation with conda worked seamlessly on a Linux cluster, as it found the bioconda channel for bcftools, htslib, pysam, samtools.

**Comment by sebastian-luna-valero** [comment]
Indeed, please test with:

```
conda install umi_tools --override-channels --channel https://conda.anaconda.org/toms --channel defaults --channel https://conda.anaconda.org/conda-forge --channel https://conda.anaconda.org/r --channel https://conda.anaconda.org/bioconda --yes
```

Best regards,
Sebastian.

**Comment by davismcc** [comment]
Yep, that works like a charm - installs perfectly. Thanks!

Davis

**Comment by IanSudbery** [comment]
I can only think that samtools, htslib and bcftools are being pulled down as dependencies of pysam - we certainly don't use them. And pysam per-se doesn't require them either - it depends how it is built. Perhaps its something to do with a new version of pysam? 

@TomSmithCGAT When you did the installation tests on OSX is it possible you had the bioconda channel already active?

**Comment by TomSmithCGAT** [comment]
@IanSudbery. These are actually erroneously included in the requirements for umi_tools :anguished:

The conda version has fallen somewhat behind the latest github. I had been holding it off until I had resolved #54 so we could release the an update including the new _group_ command but this is taking longer to finalize than I'd hoped - I'll update umi_tools in conda and pip tomorrow and remove these dependencies.

**Comment by TomSmithCGAT** [comment]
Conda and pypi versions have now been updated to 0.3.3 for py2.7, py3.4 & py3.5 with the correct dependencies. See https://github.com/CGATOxford/UMI-tools/tree/0.3.3
